tests/unit/modules/network/nxos/storage/test_nxos_zone_zoneset.py

```diff
@@ -705,3 +695,71 @@ def test_zoneset_activate_deactivate_2(self):
                 "no terminal dont-ask",
             ],
         )
+
+    def test_bug_zone_remove(self):
+        mem1 = {"pwwn": "21:01:00:1b:32:a1:c0:a8", "remove": True}
+        mem2 = {"pwwn": "50:06:01:6a:47:e4:6e:59", "remove": True}
+        a = dict(
+            zone_zoneset_details=[
+                dict(
+                    vsan=221,
+                    zone=[dict(name="zv221", members=[mem1, mem2])],
+                )
+            ]
+        )
+        set_module_args(a, True)
+        self.execute_show_cmd_zone_status.return_value = load_fixture(
+            "nxos_zone_zoneset", "shzonestatus_4.cfg"
+        )
+        self.execute_show_cmd_zone.return_value = load_fixture(
+            "nxos_zone_zoneset", "shzone_2.cfg"
+        )
+        self.execute_show_cmd_zoneset_active.return_value = load_fixture(
+            "nxos_zone_zoneset", "shzonesetactive_0.cfg"
+        )
+        result = self.execute_module(changed=True)
+        self.assertEqual(
+            result["commands"],
+            [
+                "terminal dont-ask",
+                "zone name zv221 vsan 221",
+                "no member pwwn 21:01:00:1b:32:a1:c0:a8",
+                "no member pwwn 50:06:01:6a:47:e4:6e:59",
+                "zone commit vsan 221",
+                "no terminal dont-ask",
+            ],
+        )
+
+    # def test_bug_from_active_zoneset_add(self):
```

Comment:
This unit test seems to be commented out. Can you add it back?

Reply:
the testcase was changed, added the changed test case